### PR TITLE
Upgraded antora-preview to latest version

### PR DESCRIPTION
**Pull request by akosma**
This PR provides upgrades antora-preview to the last version (2.3.4) which includes a new live reloading capability:

1. Run `make docs-serve`.
2. Open http://localhost:2020 on a browser with the [LiveReload plugin](https://github.com/vshn/antora-preview#livereload).
3. Edit your documentation in your preferred editor.
4. Watch your changes appear automatically on the browser.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [ ] Update the documentation.
- [ ] Update tests.
- [ ] Update the ./CHANGELOG.md.
- [ ] Link this PR to related issues.

### CODE REVIEW

**Comment by srueg** [comment]
Unfortunately it doesn't seem to work as expected. My changes are not recompiled (even when manually reloading the browser). Might be because I'm using Docker for macOS where the actual Docker daemon is running in a VM and issues with fsnotify are common?

**Comment by srueg** [comment]
Maybe adding `--force-polling` to the guard cmd could help?

**Comment by akosma** [comment]
@srueg could you please try again? (I've re-pushed the vshn/antora-preview:2.3.4 image with the force pulling option)

**Comment by corvus-ch** [comment]
> Unfortunately it doesn't seem to work as expected.

It did work for me (docker for mac 2.3.6.0). No idea if my test before or after adding the `--force-polling` option.

**Comment by akosma** [comment]
@corvus-ch your tests did _not_ include the `--force-polling` option, which is a CPU hog and I wanted to avoid at all costs. I modified the image including `libnotify` which is reported in other projects to help in the case of running these scenarios on Docker for Mac.

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ docker_cmd  ?= docker
 docker_opts ?= --rm --tty --user "$$(id -u)"
 
 vale_cmd           ?= $(docker_cmd) run $(docker_opts) --volume "$${PWD}"/docs/modules/ROOT/pages:/pages vshn/vale:2.1.1 --minAlertLevel=error /pages
-antora_preview_cmd ?= $(docker_cmd) run --rm --publish 2020:2020 --volume "${PWD}":/antora vshn/antora-preview:2.3.3 --style=syn --antora=docs
+antora_preview_cmd ?= $(docker_cmd) run --rm --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora vshn/antora-preview:2.3.4 --style=syn --antora=docs
```
Comment (srueg) [inline Makefile:7]:
```suggestion
antora_preview_cmd ?= $(docker_cmd) run $(docker_opts) --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora vshn/antora-preview:2.3.4 --style=syn --antora=docs
```

I run into permission issues on Linux.

Comment (srueg) [reply Makefile:7]: Now other permission issues arise:
```
06:36:43 safelyRenameFile [ERRO] Failed copying from /tmp/temp679182690 to /preview/playbook.yml
```

**Pull request closed without merging**